sdk/src/main/java/com/hedera/hashgraph/sdk/HederaTrustManager.java

```diff
@@ -29,16 +35,18 @@ public void checkClientTrusted(X509Certificate[] chain, String authType) throws
     @Override
     public void checkServerTrusted(X509Certificate[] chain, String authType) throws CertificateException {
         if (certHash == null) {
+            logger.warn("skipping certificate check since no cert hash was found");
```

Comment:
I think instead of just a log, a new option `Client.setVerifyCertificate()` or similar would be more explicit. By default it should be true and throw an error if no cert hash. By setting it to false, people could skip verification check and it would also allow the verification check to work for pre-prod environments.